tests/common/macros.rs

```diff
@@ -46,3 +46,22 @@ macro_rules! path_concat {
         pb.to_str().unwrap().to_owned()
     }};
 }
+
+#[macro_export]
+macro_rules! util_name {
+  () => ( module_path!().split("_").nth(1).expect("no test name") )
+}
+
+#[macro_export]
+macro_rules! new_ucmd {
+  () => ( TestScenario::new(util_name!()).ucmd() )
+}
+
+#[macro_export]
+macro_rules! at_and_ucmd {
+  () => ({
+    let ts = TestScenario::new(util_name!());
+    let mut ucmd = ts.ucmd();
+    (ts.fixtures, ucmd) 
```

Comment:
because fixtures is a struct data member of TestScenario, so providing it requires a move or use of lifetimes. Alternatively, we could just rename it ```ucmd_and_at``` and flip the bindings wherever its used